disasm-test/Main.hs

```diff
@@ -578,3 +735,14 @@ rmFile tmp = do Keep keep <- gets keepTemp
                   $ do Details dets <- gets showDetails
                        when dets $ liftIO $ putStrLn $ "## Removing " <> tmp
                        liftIO $ removeFile tmp
+
+knownBugs :: [ (FilePath, String) ]  -- FilePath is TS.rootMatchName
+knownBugs =
+  let pr223pr228 = "unnamed metadata indices are currently duplicated (see https://github.com/GaloisInc/llvm-pretty-bc-parser/pull/223 and https://github.com/GaloisInc/llvm-pretty-bc-parser/pull/228)"
```

Comment:
Consider breaking this up into multiple lines for easier readability.

Reply:
Done.